here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -627,4 +628,12 @@ <h2 class="text-2xl md:text-3xl benfritz text-white text-center">
     });
   }
 </script>
+<script>
```

Comment:
JS should not be in the body/template/view but rather in a separate JS file please.